### PR TITLE
[AdminBundle] Prevent duplicate reinits in same pagepart (reorder)

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

### CODE REVIEW

**Comment by jockri** [comment]
@diskwriter Can you take a look?

**Comment by diskwriter** [comment]
Hi Danny

I just noticed there's some duplicate code in this file that shouldn't be there.
Could you please remove the reInit functionality from the $.ajax success handler (https://github.com/insiders/KunstmaanBundlesCMS/blob/bugfix/prevent-reinit-pagepart/src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js#L96) and call the reInit method from there too?

I've also commented for a small improvement.

Thanks in advance

File: src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js [hunk 1]
```diff
@@ -300,11 +302,18 @@ kunstmaanbundles.pagepartEditor = function (window) {
...
+                        if ($.inArray(modules[i], uniqueModules) === -1) {
+                            uniqueModules.push(modules[i]);
+                        }
                     }
                 }
             }
         });
+
+        for (var i = 0; i < uniqueModules.length; i++) {
+            kunstmaanbundles[uniqueModules[i]].reInit();
```
Comment (diskwriter) [inline src/Kunstmaan/AdminBundle/Resources/ui/js/_pagepart-editor.js]: No real need for an extra for loop here. You could also reInit the modules in the first for loop.

**Comment by dannyvw** [comment]
Pull request is updated

**Comment by diskwriter** [comment]
Thanks, @dannyvw 
Could you also rebase your commits? Then I can merge. Thanks!

**Comment by dannyvw** [comment]
@diskwriter Done

**Comment by diskwriter** [comment]
@dannyvw Thanks!

**Pull request merged**